pkg/gadgets/snapshot/process/tracer/gadget.go

```diff
@@ -64,6 +66,26 @@ func (g *GadgetDesc) EventPrototype() any {
 	return &types.Event{}
 }
 
+func (g *GadgetDesc) OutputFormats() (gadgets.OutputFormats, string) {
+	return gadgets.OutputFormats{
+		"tree": gadgets.OutputFormat{
+			Name:        "Tree",
+			Description: "A pstree like output",
+			Transform: func(data any) ([]byte, error) {
+				processes := data.([]*types.Event)
```

Comment:
Could you please add a type check here as well and return an error, if it's not `[]*types.Event`?